data/download_all.zsh

```diff
@@ -1,17 +1,22 @@
 #!/bin/zsh -xe
 
 declare -A short_names
-short_names["US"]=USA
+short_names["United States"]=USA
 short_names["United Kingdom"]=UK
+short_names["New Zealand"]=NZ
 
 python3.7 -m venv --system-site-packages venv && source venv/bin/activate
 pip install $1/
-for country in Germany Iran Italy Spain "United Kingdom" US
+
+countries=("United Kingdom" Korea Austria Iceland Jordan Switzerland Croatia Australia Canada
+  Germany Israel Italy Malaysia "New Zealand" Spain Belgium Chile Czechia Latvia
+  Lithuania Netherlands Norway Portugal US)
```

Comment:
You are using `US` here and not `United States`, not sure if intentional.

Reply:
Thanks for noticing, that's a bug.